impl/src/util.rs

```diff
@@ -49,20 +51,33 @@ static SUPPORTED_PLATFORM_TRIPLES: &[&str] = &[
   "x86_64-unknown-freebsd",
 ];
 
+#[derive(Debug, PartialEq, Eq, Copy, Clone)]
```

Comment:
Haven't done a full review yet but are there any paths where this might require `Serialize` and `Deserialize` as well?

Reply:
Specifically asking in regards to https://github.com/google/cargo-raze/pull/415 but that's likely out of scope for that change and something I'll go update 😄

Reply:
https://github.com/google/cargo-raze/pull/282#issuecomment-825969996